parm/atm/obs/lists/gdas_prototype.yaml.j2

```diff
@@ -0,0 +1,16 @@
+observers:
```

Comment:
This list is different than the list in `snowDA`.  The `observers` is kept in the parent yaml, while this list is a list of the included obs files.
There should be consistency in the `OBS_LIST`.

Reply:
`parm/snow/obs/lists/gdas_snow.yaml.j2` contains
```
observers:
{% filter indent(width=2) %}
{% include 'snow/obs/config/adpsfc_snow.yaml.j2' %}
{% include 'snow/obs/config/snocvr_snow.yaml.j2' %}
{% set cycle = current_cycle | strftime("t%Hz") %}
{% if cycle in ['t18z'] %}
{% include 'snow/obs/config/ims_snow.yaml.j2' %}
{% endif %}
{% endfilter %}
```
`observers` is part of the snow observation list.  This pattern was followed for atmosphere observation lists.

Reply:
you are right.  I don't know what I was thinking.